tex/chProgramming.tex

```diff
@@ -2144,9 +2230,13 @@ \section{Exercises}
 \begin{coqout}{}{width=4cm}
 $~$
      = 4 : nat
-     = true : bool
+     = false : bool
      = false : bool
 \end{coqout}
+% [DG] I get the "Error: A left-recursive notation must have an explicit level."
```

Comment:
The [ssrfun](https://math-comp.github.io/math-comp/htmldoc/mathcomp.ssreflect.ssrfun.html) module contatins the following:

```Coq
(* Notations for pair/conjunction projections *)
Notation "p .1" := (fst p)
  (at level 2, left associativity, format "p .1") : pair_scope.
```